README.md

```diff
@@ -2,4 +2,17 @@ Shy'Comics
 ==========
 
 Let's start the new version of Shy'Comics!
-http://shycomics.fr/ShyComicsReturns/
\ No newline at end of file
+http://shycomics.fr/ShyComicsReturns/
+
+All the code and commit commentary must be in english.
+
+How to run this website localy
+* Clone the GitHub project into your folder named ShyComicsReturns in your Web environement folder.
+* If you have to change the name of the folder, you can change the BASE_URL in application/configs/main.json to your folder's name.
```

Comment:
Strange to tell the user to update a file he doesn't have yet. This raw should be after the one telling to copy/past `main.json.default`